.github/dependabot.yml

```diff
@@ -1,55 +1,49 @@
 version: 2
 updates:
-- package-ecosystem: cargo
-  directories:
-    - "/"
-    - "/staging/"
-  schedule:
-    interval: weekly
-  allow:
-  - dependency-type: direct
-  - dependency-type: indirect
-  groups:
+  - package-ecosystem: cargo
+    directories:
+      - /
+      - /staging/
+    schedule:
+      interval: weekly
```

Comment:
It looks like 2 space indentation added in the all file, are that really needed?

Reply:
Also, why removing "" ?

Please do change incrementally (several commits, explaining why), but only if really needed.

Reply:
I think it's `prettier` get in the way, I will turn that auto-formatting off 😥

Reply:
resolved 😊